docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -47,23 +47,26 @@ POST /_snapshot/my_backup/snapshot_1/_restore
 {
   "indices": "index_1,index_2",
   "ignore_unavailable": true,
-  "include_global_state": true,
+  "include_global_state": true,              <1>
```

Comment:
I really think we should use `false` in the example here and adjust the callout to match, with the expectation that people will continue to copy-paste this anyway.

```suggestion
  "include_global_state": false,              <1>
```